docs/rules/prefer-query-selector.md

```diff
@@ -7,7 +7,7 @@
 <!-- end auto-generated rule header -->
 <!-- Do not manually modify this header. Run: `npm run fix:eslint-docs` -->
 
-It's better to use the same method to query DOM elements.
+It's better to use the same method to query DOM elements. This helps keep consistency and it lends itself to future improvements (e.g. more specific selectors).
```

Comment:
Best I could do. The rule is more about style than utility (`getElementsByTagName` looks obsolete).